pippo-server-parent/pippo-jetty/src/main/java/ro/pippo/jetty/JettyServer.java

```diff
@@ -58,46 +55,8 @@
 
     private Server server;
 
-    private final ExecutorService executor;
-    private final CountDownLatch  startLatch;
-
-    public JettyServer() {
```

Comment:
I put some logging (not checked in) to confirm the AbstractWebServer constructor is indeed still being called.